internal/provisioner/plain/README.md

```diff
@@ -166,50 +172,88 @@ spec:
         type: image
 EOF
+```
+
+A message saying the `BundleInstance` is created should be returned:
+
+```text
 bundleinstance.core.rukpak.io/combo created
 ```
 
-Check the Bundle status via `kubectl get bundle -l app=combo`. Eventually the Bundle should show up as Unpacked.
+Next, check the Bundle status via:
 
 ```console
-$ kubectl get bundle -l app=combo
+kubectl get bundle -l app=combo
+```
+
+Eventually the Bundle should show up as Unpacked:
+
+```text
 NAME          TYPE   PHASE      AGE
 combo-9njsj   image  Unpacked   10s
 ```
 
-Check the BundleInstance status to ensure that the installation was successful.
+Check the BundleInstance status to ensure that the installation was successful:
 
 ```console
-$ kubectl get bundleinstance combo
+kubectl get bundleinstance combo
+```
+
+A successful installation will show InstallationSucceeded as the `INSTALL STATE`:
+
+```text
 NAME    INSTALLED BUNDLE   INSTALL STATE           AGE
 combo   combo-9njsj        InstallationSucceeded   10s
 ```
 
-From there, check out the combo operator deployment and ensure that the operator is present on the cluster.
+From there, check out the combo operator deployment and ensure that the operator is present on the cluster:
 
 ```console
+# Check the combo operator deployment
+kubectl -n combo get deployments.apps combo-operator
+
+# Check that the operator is present
+kubectl -n combo get deployments.apps combo-operator -o yaml | grep 'image:' | xargs
+```
+
+The deployment should show that the operator is ready and available and the operator
+should be present:
+
+```text
 $ kubectl -n combo get deployments.apps combo-operator
 NAME             READY   UP-TO-DATE   AVAILABLE   AGE
 combo-operator   1/1     1            1           10s
 
 $ kubectl -n combo get deployments.apps combo-operator -o yaml | grep 'image:' | xargs
-image: quay.io/tflannag/combo:v0.0.1
+image: quay.io/operator-framework/combo-operator:v0.0.1
 ```
 
-The operator should be successfully installed.
+This means the operator should be successfully installed.
 
-Next, the `plain` provisioner continually reconciles BundleInstance resources. Let's try deleting the combo deployment:
+The `plain` provisioner continually reconciles BundleInstance resources. Next, let's try deleting the combo deployment:
 
 ```console
+kubectl -n combo delete deployments.apps combo-operator
+```
+
+A message saying the deployment was deleted should be returned:
+
+```text
 $ kubectl -n combo delete deployments.apps combo-operator
 deployment.apps "combo-operator" deleted
 ```
 
-Check for the deployment again, it will be back on the cluster. The provisioner ensures that all resources required for
-the BundleInstance to run are accounted for on-cluster.
+The provisioner ensures that all resources required for the BundleInstance to run are accounted for on-cluster.
+So if we check for the deployment again, it will be back on the cluster:
+
+```text
+$ kubectl -n combo get deployments.apps combo-operator
```

Comment:
nit: Should probably decide if we want to have the `$` on a lot of these as it seems like it is inconsistent. 
```suggestion
kubectl -n combo get deployments.apps combo-operator
```

Reply:
I tried to split up the commands and the console output so the user can use the copy function in github to quickly grab the command. I've also tried to make the color coding consistent so it's more obvious which is which, but I didn't go back and make it consistent across the whole doc so I'm going to do another pass to just make sure it all flows.